{{cookiecutter.app_name}}/.github/workflows/lint_docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Lint OpenAPI docs
+
+on: pull_request
+
+concurrency:
+  group: {{ "${{ github.workflow }}-${{ github.ref }}" }}
+  cancel-in-progress: true
+
+jobs:
+  docs_lint:
+    name: Run lint for API docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up node and restore cached dependencies
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18.x"
```

Comment:
We should update generated README to include node requirements for all types of projects. In current README we have:
```
{% if cookiecutter._web_variant == "yes" %}- [Node - 14](https://nodejs.org/en/){% endif %}
```

Reply:
https://github.com/nimblehq/gin-templates/pull/106/commits/7dbb08f0961586fa911ee2ebe4544fb657083cc6

`{% if cookiecutter._web_variant == "yes" %}- [Node - 14](https://nodejs.org/en/){% endif %}` line should be removed, otherwise for web variant it will generate:
- [Node - 18]
- [Node - 14]